apps/vaporgui/PCheckbox.cpp

```diff
@@ -0,0 +1,20 @@
+#include "PCheckbox.h"
+#include "VLineItem.h"
+#include <vapor/ParamsBase.h>
+
+PCheckbox::PCheckbox(const std::string &tag, const std::string &label)
+: PWidget(tag, new VLineItem(label==""?tag:label, _qcheckbox = new QCheckBox))
```

Comment:
How is `_qcheckbox` destroyed if it is created without specifying any parent/child relationship?

Reply:
Oops, got it, sorry!